DOCU-2908 Endpoint deprecation note and instruction (#5032)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/analytics/index.md

  * If a user does not have access to Vitals data, charts will not display.

## Vitals API

> Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC. The Vitals API is described using the Open API Specification, you can view it by downloading the [`vitals.yaml`](/api/vitals.yaml) file. You can use the API to retrieve health and traffic information from your cluster.

Reviewer: ```suggestion
Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC. 

The Vitals API is described using the Open API Specification. You can view it by downloading the [`vitals.yaml`](/api/vitals.yaml) file. Use the API to retrieve health and traffic information from your cluster.
```
Got rid of the second "you can" for better flow.

For example, you can use the API to retrieve the number of requests for a service in a workspace by issuing the following `GET` request: 

```bash
curl -i -X GET http://localhost:8001/WORKSPACE_NAME/vitals/status_codes/by_service \
    -d "service_id=030f4ade-69ed-48b0-b3e1-cd088a1f2a09" \
    -d "interval=minutes"
```

> A succesful request will return a payload with the following data:

Author: ```suggestion
A successful request will return a payload with the following data:
```

> A successful request will return a payload with the following data:

Reviewer: ```suggestion
A successful request returns a payload with the following data:
```

> ```bash

Reviewer: ```suggestion
```json
```

  * If a user does not have access to Vitals data, charts will not display.

## Vitals API

> Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC. 

Reviewer: ```suggestion
Vitals data is available using the Vitals API. Access to these endpoints can be managed using RBAC. 
```

The Vitals API is described using the Open API Specification. You can view it by downloading the [`vitals.yaml`](/api/vitals.yaml) file. Use the API to retrieve health and traffic information from your cluster.

For example, you can use the API to retrieve the number of requests for a service in a workspace by issuing the following `GET` request: 

```bash
curl -i -X GET http://localhost:8001/WORKSPACE_NAME/vitals/status_codes/by_service \

>     -d "service_id=030f4ade-69ed-48b0-b3e1-cd088a1f2a09" \

Reviewer 2: Are we usually replacing "real" values with some placeholders here? I am not sure.

Author: We are moving in that direction to signal to people where they would replace their values, and also to make it clear that the code can't just be copied and ran.
